src/com/owncloud/android/lib/resources/activities/GetRemoteActivitiesOperation.java

```diff
@@ -59,7 +59,7 @@
     private static final String TAG = GetRemoteActivitiesOperation.class.getSimpleName();
 
     // OCS Routes
-    private static final String OCS_ROUTE = "/apps/activity/api/v2/activity";
+    private static final String OCS_ROUTE = "/ocs/v2.php/apps/activity/api/v2/activity";
```

Comment:
@AndyScherzinger for you, /ocs/v1.php/cloud/activity?format=json is used afaik :D